### PR TITLE
Document affected third parties by a breaking change

### PR DESCRIPTION
**Pull request by tsusanka**
As discussed with @slush0 it seems as a good idea to have this list also here.

I have added the ones @prusnak mentioned and also added all parties that had over 100 stars on [this list](https://github.com/trezor/connect/network/dependents?package_id=UGFja2FnZS01MTA2MzE4OQ%3D%3D).

### CODE REVIEW

**Comment by tsusanka** [comment]
And two more to consider from trezor-firmware [dependents list](https://github.com/trezor/trezor-firmware/network/dependents?dependents_after=MTA4NjM2Mzc0OTU):

- https://github.com/AXErunners/electrum-axe
- https://github.com/kayagoban/shadowlands

File: docs/misc/third-parties.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+That need to be notified when a protocol breaking change occurs.
+
+### Using trezorlib:
+_This usually requires some code changes in the affected software._
+
+- Electrum https://github.com/spesmilo/electrum
+- HWI https://github.com/bitcoin-core/HWI
+- Trezor Agent https://github.com/romanz/trezor-agent
+- Monero https://github.com/monero-project/monero
```
Comment (matejcik) [inline docs/misc/third-parties.md:11]: monero is not using trezorlib

Comment (tsusanka) [reply docs/misc/third-parties.md]: Fixed.

File: docs/misc/third-parties.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+### Using no Trezor libraries
+- Monero https://github.com/monero-project/monero
+- Mycelium Android https://github.com/mycelium-com/wallet-android
+- Mycelium iOS https://github.com/mycelium-com/wallet-ios
+- Blockstream Green Android https://github.com/Blockstream/green_android
+- Blockstream Green iOS https://github.com/Blockstream/green_ios
+
+### Using Connect:
+_This usually only requires updating Connect to the latest version._
+
```
Comment (prusnak) [inline docs/misc/third-parties.md:21]: Let's add a link to https://github.com/trezor/connect/network/dependents

**Comment by matejcik** [comment]
do we also want an explicit list of HWI users? (wasabi comes to mind)

File: docs/misc/third-parties.md (hunk 2 above)
Comment (tsusanka) [reply docs/misc/third-parties.md:21]: Added.

**Comment by tsusanka** [comment]
@matejcik we might, but does it concern Wasabi if it gets solved by HWI? Or it can't be generalized like that?

**Comment by matejcik** [comment]
similar to Connect, Wasabi will need to at least bump their HWI version.

**Comment by tsusanka** [comment]
Good point. It makes sense then. Any other that might use HWI? cc @prusnak

**Comment by prusnak** [comment]
I know of BTCPay and Wasabi.

**Comment by tsusanka** [comment]
Added.

---


Also after merging this I will open issues on those projects, unless those where @prusnak already did it of course.

**Comment by tsusanka** [comment]
I will merge this tonight unless there some other comments.

**Pull request merged**